config.py

```diff
@@ -12,11 +12,9 @@ class DataSourceConfig:
     tmp_folder: str
     minio_path: str
     file_name: str
-    resource_id: str | None = None
-    url: str | None = None
+    files_to_download: dict[str, dict[str, str]]
```

Comment:
I think this new var would benefit from some documentation. What's in the dict and what's expected.

Reply:
True! 💯 I will work on it

Reply:
Done! Can you tell me what you think about it? 😄

Reply:
Looks good to me 🚀